src/core/basetypes/MCData.cpp

```diff
@@ -664,23 +691,22 @@ Data * Data::decodedDataUsingEncoding(Encoding encoding)
                 if (length == 0) {
                     break;
                 }
-                decoded_length = uudecode(current_p, length);
+                decoded_length = uudecode(current_p, length, decoded_buf, sizeof(decoded_buf));
```

Comment:
I work on iOS imap client. iOS app has very strong memory constraints. The app may be killed if it uses a lot of memory. UUE is very unpopular encoding format, but I think it is very bad explanation for user why application was crashed.